src/rt/arc.rs

```diff
@@ -90,6 +102,10 @@ impl Arc {
         self.obj.branch(Action::RefDec);
 
         rt::execution(|execution| {
+            if execution.log {
+                println!("Arc::ref_dec");
```

Comment:
Similarly, maybe this should include the ref count after decrement?

Reply:
Done.